src/utils/mutations.ts

```diff
@@ -752,6 +888,88 @@ export async function confirmRepeatable(
 	})
 }
 
+export async function confirmAllRepeatables(tasks: RepeatableCompletion[], classID: string) {
```

Comment:
similarly, if you have time try to reduce the repetition with confirmRepeatable just to make this easier to modify/debug in the future